index.bs

```diff
@@ -681,14 +706,551 @@ id=] must be the same as the [=window handle=].
+
+            1. [=Set=](|serialized|, "localName", [=Get=](|value|, "localName"), false)
+
+            1. [=Set=](|serialized|, "nodeValue", [=Get=](|value|, "nodeValue"), false)
+
+            1. Let |child node count| be the size of |serialized|'s [=children=].
+
+            1. [=Set=](|serialized|, "childNodeCount", |child node count|, false)
+
+            1. If |max depth| is null and or equal to 0 let |children| be null.
+               Otherwise, let |children| be an empty list and, for each node
+               |child| in the [=children=] of |value|:
+
+              1. Let |serialized| be the result of [=serialize as a remote value=]
+                 with |child|, |max depth| - 1, |node details| and
+                 |set of known objects|.
+
+              1. Append |serialized| to |children|.
+
+            1. [=Set=](|serialized|, "children", |serialized|, false)
+
+            1. Let |attributes| be a new [=Object=].
+
+            1. For each |attribute| in |value|'s <a spec=dom>attributes</a>:
+
+              1. Let |name| be |attribute|'s [=Attr/qualified name=]
+
+              1. Let |serialized| be |attribute|'s [=Attr/value=].
+
+              1. [=Set=](|attributes|, |name|, |serialized|, false)
+
+            1. [=Set=](|serialized|, "attributes", |attributes|, false)
+
+        1. Let |remote value| be an [=object=] matching the <code>NodeValue</code>
+           production in the [=local end definition=], with the <code>objectId</code>
+           property set to the [=object id for an object=] |value|, and <code>value</code>
+           set to |serialized|, if |serialized| is not null.
+
+      <dt>|value| is a [=platform object=] that implements [=WindowProxy=]
+      <dd>1. Let |remote value| be an [=object=] matching the <code>WindowProxyValue</code>
+             production in the [=local end definition=], with the <code>objectId</code>
+             property set to the [=object id for an object=] |value|.
+
+      <dt>|value| is a [=platform object=]
+      <dd>1. Let |remote value| be an [=object=] matching the <code>ObjectValue</code>
+             production in the [=local end definition=], with the <code>objectId</code>
+             property set to the [=object id for an object=] |value|.
+
+      <dt>[=IsCallable=](|value|)
+      <dd>Let |remote value| be an [=object=] matching the <code>FunctionValue</code>
+          production in the [=local end definition=], with the <code>objectId</code>
+          property set to the [=object id for an object=] |value|.
+
+      <dt>Otherwise:
+      <dd>
+      1. let |serialized| be null.
+
+      1. If |value| is not in the |set of known objects|, and |max depth|
+         is not null and greater than 0, run the following steps:
+         1. Append |value| to the |set of known objects|
+
+         1. Let |serialized| be the result of [=serialize as a mapping=] given
+            [=EnumerableOwnPropertyNames=](|value|, key+value), |max depth|, |node
```

Comment:
key+value looked suspect, but looks like the right way to do invoke [EnumerableOwnPropertyNames](https://tc39.es/ecma262/#sec-enumerableownpropertynames) 👍

But should there be some assert right before this, since we're assuming that *value* is an Object here?